charts/sourcegraph/templates/priorityClass.yaml

```diff
@@ -0,0 +1,11 @@
+{{- range $class := .Values.priorityClasses }}
```

Comment:
maybe add this to values.yaml with a default of empty slice?

- we can make it show up in the generated docs 
- in controller, we fetch the values.yaml and run test to marshal/unmarshal it with our own structs